aws/resource_aws_spot_fleet_request_test.go

```diff
@@ -620,7 +620,7 @@ func TestAccAWSSpotFleetRequest_withTags(t *testing.T) {
 	})
 }
 
-func TestAccAWSSpotFleetRequest_placementTenancy(t *testing.T) {
+func TestAccAWSSpotFleetRequest_placementTenancyAndGroup(t *testing.T) {
```

Comment:
Nit: It is generally preferable to create new testing rather than modify existing tests to ensure we do not introduce regressions (even if it is very verbose to do so). 👍